assignments/callbacks.js

```diff
@@ -23,25 +23,49 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
```

Comment:
good job on all the callbacks. You seem to have a grasp on how to execute on them and on their purpose, that's what I like to see.